### ISSUE TITLE
Object operator color

### ISSUE THREAD

**Issue by robbinjohansson** [sethlopezme/InspiredGitHub.tmtheme#9]
👋 Hey, 

Nice scheme! I was wondering, shouldn't the object operator in php be red-ish like in github? Any idea why mine isn't? (it's just regular black/text color)

**Actual:**

<img width="946" alt="screen shot 2018-09-16 at 16 29 54" src="https://user-images.githubusercontent.com/23704274/45597512-d18e5380-b9cd-11e8-9d7a-407c12b99e52.png">

**Expected:**
If I open your theme in a theme editor it's looking as expected, but not when I install the theme:

https://tmtheme-editor.herokuapp.com/#!/editor/url/https://raw.githubusercontent.com/sethlopezme/InspiredGitHub.tmtheme/master/InspiredGitHub.tmTheme

Thanks!

**Comment by sethlopezme** [comment]
Hey! 👋

Thanks for checking out my color scheme and taking the time to report an issue! Unfortunately, this repository isn't actively maintained anymore. I just don't use Sublime Text these days. I know that can be frustrating to read, especially after seeking out this repository and filing an issue. I am, however, willing to accept a pull request if you decide to fix the issue you reported.

After seeing your issue, I realized that the maintenance status of this repository isn't stated in the README, so I've updated it to let others know, as well. Again, thanks for checking out my theme and, aside from the reported issue, I hope you've enjoyed it.
